scripts/run-component-tests.js

```diff
@@ -38,36 +39,51 @@ shelljs.set('-e');
 shelljs.cd(projectDir);
 
 // Extracts the supported command line options.
-const {_: components, local, firefox, watch} = minimist(args, {
-  boolean: ['local', 'firefox', 'watch'],
+const {_: components, local, firefox, watch, all} = minimist(args, {
+  boolean: ['local', 'firefox', 'watch', 'all'],
   default: {watch: true},
 });
 
-// Exit if no component has been specified.
-if (!components.length) {
-  console.error(chalk.red(
-      'No component specified. Specify a component name, or pass a ' +
-      'path to the component directory.'));
-  process.exit(1);
-}
-
 // We can only run a single target with "--local". Running multiple targets within the
 // same Karma server is not possible since each test target runs isolated from the others.
-if (local && components.length > 1) {
+if (local && (components.length > 1 || all)) {
   console.error(chalk.red(
       'Unable to run multiple components tests in local mode. ' +
       'Only one component at a time can be run with "--local"'));
   process.exit(1);
 }
 
-const bazelBinary = watch ? 'ibazel' : 'bazel';
-const bazelAction = local ? 'run' : 'test';
+const bazelBinary = `yarn -s ${watch ? 'ibazel' : 'bazel'}`;
 const testTargetName =
     `unit_tests_${local ? 'local' : firefox ? 'firefox-local' : 'chromium-local'}`;
+
+// If the `--all` flag has been specified, we run tests for all components in the
+// repository. The `--firefox` flag can be still specified.
+if (all) {
+  if (components.length) {
+    console.error(
+        chalk.red('Script cannot be run with `--all` if individual components are specified.'));
+    process.exit(1);
+  }
+  shelljs.exec(
+      `${bazelBinary} test //src/... --test_tag_filters=-e2e,-browser:${testTargetName} ` +
+      `--build_tag_filters=-browser:${testTargetName} --build_tests_only`);
+  return;
+}
+
+// Exit if no component has been specified.
+if (!components.length) {
+  console.error(chalk.red(
+      'No component specified. Specify a component name, or pass a ' +
```

Comment:
or use --all?

Reply:
👍 improved the message, and added a few command examples (in case someone just runs `yarn test`)